src/utils/daos_autotest.c

```diff
@@ -27,8 +27,15 @@
 /** Input arguments passed to daos utility */
 struct cmd_args_s *autotest_ap;
 
+/* The start and end time in ns for individual test*/
+uint64_t	t_start, t_end;
+char		iops[32];
 /** How many concurrent I/O in flight */
 #define MAX_INFLIGHT 16
+#define OUTPUT_IOPS(x) ({\
+		snprintf(iops, sizeof(iops)-1, "%6.1lfK IO/sec", (x));\
```

Comment:
%6.1 seems little bit short as it will overflow for 10M IO/sec.
Not sure of the max value expected with this tool.
From my side, the best solution should be to print in a "humanize" format (if possible).